### ISSUE TITLE
Pages missing in PDF export

### ISSUE THREAD

**Issue by nouyang** [xournalpp/xournalpp#879]
When I export to PDF from a xournalpp file, sometime pages go missing. I've included a mostly minimal example.

The xournalpp source files:
[xournalpp_pdf_missing_pages.zip](https://github.com/xournalpp/xournalpp/files/2865974/xournalpp_pdf_missing_pages.zip)

The exported files:
[xournalpp_NOBUG.pdf](https://github.com/xournalpp/xournalpp/files/2865975/xournalpp_NOBUG.pdf)
[xournalpp_PDFBUG.pdf](https://github.com/xournalpp/xournalpp/files/2865976/xournalpp_PDFBUG.pdf)

My versions:
```
sudo add-apt-repository ppa:andreasbutti/xournalpp-master
xournalpp is already the newest version (1.0.6-0~201902071732~ubuntu18.10.1).
```

My console (the printouts I get on opening are the same for the "PDFBUG" and "NOBUG" versions):

```sh
$ xournalpp
ALSA lib pcm.c:2565:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2565:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2565:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
ALSA lib pcm_route.c:869:(find_matching_chmap) Found no matching channel map
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock

** (xournalpp:30872): WARNING **: 11:44:56.359: lastSavePath is not set!
** Message: 11:45:03.853: Filename: /home/xournalpp_PDFBUG.xopp

** (xournalpp:30872): WARNING **: 11:45:03.908: Parser: attribute texlength not found!

** (xournalpp:30872): WARNING **: 11:45:03.909: Parser: attribute texlength not found!

** (xournalpp:30872): WARNING **: 11:45:03.909: Parser: attribute texlength not found!

** (xournalpp:30872): WARNING **: 11:45:03.915: Parser: attribute texlength not found!

** (xournalpp:30872): WARNING **: 11:45:03.916: Parser: attribute texlength not found!

** (xournalpp:30872): WARNING **: 11:45:03.916: Parser: attribute texlength not found!
** Message: 11:45:12.236: MemoryLeak: sum 0 objects.
```

**Comment by nouyang** [comment]
Hold on here, there is actually secret hidden latex in that file that I can't see in the GUI. This latex is not exported to either the PDF or the PNG exports, no matter if there's an "empty" last page or not.

I think if I can remove the invisible latex through a game of drawing random selection rectangles, it might export correctly... but I can't find the last one.... and now my hand-drawn graph doesn't show in the PNG export either. :'( 

You can see the graph is you select it, it's in the bottom half of the page:

![image](https://user-images.githubusercontent.com/764681/52804147-7bdbd700-3051-11e9-8931-22432b197712.png)


Alas I will have to give up on finding the Last Latex for now, but here is the xopp zip'd and attached:
[xournalpp_PDFBUG.zip](https://github.com/xournalpp/xournalpp/files/2866056/xournalpp_PDFBUG.zip)

**Comment by LittleHuba** [comment]
Sounds a little bit like the selection bug in #840. Maybe the have something in common?
Still the exporter should fail more gracefully and output everything it can.

**Comment by bhennion** [comment]
Closing this issue. It seems to have been caused by a corrupted .xopp file, with a size 0 TexImage.
Whatever created this corrupted file should be fixed by now (see #2464 and #2857)
